client/post-editor/editor-gutenberg-opt-in-notice/style.scss

```diff
@@ -0,0 +1,7 @@
+/** @format */
+
+.editor-gutenberg-opt-in-notice {
```

Comment:
With this notice relative to content, it's pretty easy to scroll out of view. Maybe consider fixed positioning here.

cc @shaunandrews

Reply:
I played with this a bit and I can't figure out how to get `position: fixed` to work with horizontal centering and adapting to the page width. For example, at this width it gets cut off. Before:

![screen shot 2018-10-25 at 23 54 23](https://user-images.githubusercontent.com/52152/47549605-692a7000-d8b1-11e8-83a4-7ca26990ee78.png)

After: 
![screen shot 2018-10-25 at 23 54 31](https://user-images.githubusercontent.com/52152/47549611-6d568d80-d8b1-11e8-96d6-f67400251b75.png)

@shaunandrews can you help?

Reply:
To build on top of @mmtr suggestions, I've had good results with this:

```css
.editor-gutenberg-opt-in-notice {
	bottom: 27px;
	left: 0;
	right: 0;
	margin: 0;
	position: fixed;
}

.layout.focus-sidebar .editor-gutenberg-opt-in-notice {
	width: calc( 100% - 272px );
}
```

The notice sticks to the bottom (the `27px` offset is to account for the words count and last saved messages), and fills all the horizontal space, resizing itself whenever the sidebar is open (`.layout.focus-sidebar`, instead of `.layout.focus-content` when the sidebar is closed).

Of course feel free to change this, for example by adjusting the horizontal margins if needed as described by @mmtr.